tests/foreman/ui/test_contentview.py

```diff
@@ -300,6 +308,8 @@ def test_positive_add_puppet_module(session, module_org):
     :expectedresults: content view is created, updated with puppet module
 
     :CaseLevel: Integration
+
+    :CaseImportance: high
```

Comment:
Lowercase is completely fine, but since you need to add new commit to this PR anyway, you might change it to `High` for the sake of consistency.